src/platform/forms/exportsFile.js

```diff
@@ -123,11 +123,22 @@ import {
   getCurrentPageName,
 } from './helpers';
 
+import Subtask, {
+  SUBTASK_SESSION_STORAGE,
+  getStoredSubTask,
+  setStoredSubTask,
+  resetStoredSubTask,
+} from './sub-task';
+
 export default {
+  Subtask,
+  SUBTASK_SESSION_STORAGE,
+  getStoredSubTask,
+  setStoredSubTask,
+  resetStoredSubTask,
   states,
   militaryStates,
   militaryCities,
-  branchCodeLabels,
```

Comment:
Why is the `branchCodeLabels` export being removed?

Reply:
good catch - I didn't quite finish fixing an earlier bug - fixed in latest commit.

Reply:
Adding some context for anyone looking at this in the future. The [`branchCodeLabels` import was made in a commit on August 19](https://github.com/department-of-veterans-affairs/vets-website/pull/21977/commits/fa225953cc66d000cbba6d1743ddf553b74ee2df), then the const was [renamed in another PR on August 23](https://github.com/department-of-veterans-affairs/vets-website/pull/21976/files#diff-8fcc2b9b82bc1241bab1450166640c7888f127ff7181f0395aaa30893d8316a4R3).